tools/misc/gen_release.py

```diff
@@ -154,8 +154,11 @@ def trigger_build(self, token, project):
 
     def upload_sftp(self, artifacts, signatures, checksums):
         """Uploads artifacts to get.please.build via sftp."""
-        with open('latest_version', 'w') as f:
+        with open('latest_prerelease_version', 'w') as f:
```

Comment:
You will need to update the instructions in the batch file below as well - i.e. line 171 needs to be within a similar check for `self.is_prerelease`, etc.

Reply:
Oops, fixed.